### ISSUE TITLE
Suggestion: Link embeddings from transposed table

### ISSUE THREAD

**Issue by StaffanBetner** [anders-biostat/sleepwalk#9]
Let say I have e.g. a matrix where I have individuals (cells e.g.)  as rows and features as columns, and then run a UMAP on both the ordinary matrix, and the transposed one. Then it would be natural to want to look at the individual UMAP with the default usage (the distances to other individuals), but it would also be interesting to see the features for that individual (and vice versa).

Is it clear what I mean?

**Comment by simon-anders** [comment]
Yes, implementing this has been on our to-do list since quite a while. 

My idea was to make it such that the two UMAPs are displayed side by side: to the left, the panel with the cells UMAP, to the right, the one of the genes UMAP.  If the mouse hovers over a point on the left (i.e., a cell), the other points in the left panel are coloured according to their distance to the cell under the mouse cursor (i.e., the usual Sleepwalk behaviour), while the points on the right-hand panel (the genes) are coloured according to the strength of expression of the gene in the cell under the mouse.

If one moves the mouse to the right panel, the roles flip: the right panel is coloured according to the gene's correlation with the gene under the mouse (Sleepwalk style), and the left panel shows the expression of the gene under the cursor in each cell.

Should be quickly done; I hope we soon find the time to try it.

**Comment by StaffanBetner** [comment]
Fun to hear that other people have had the same idea!
